.storybook/webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const config = require( '../webpack.config' );
+config.module.rules = [
+	...config.module.rules.filter( ( rule ) => ! rule.test.test( 'file.scss' ) ),
```

Comment:
Ah, okay. We might want to add a comment explaining this. Probably enough to test `'.scss'` too.